packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -97,6 +99,13 @@ contract DependencyRegistryV0 is
         );
     }
 
+    function _onlyInRangeIndex(
+        uint256 _index,
+        uint256 _maxIndex
```

Comment:
nit: should this be called `_length` instead of `_maxIndex` (since length is maxIndex+1)?

Reply:
Good call out – updated per your suggestion in latest commit.